### PR TITLE
Issue 3 - Display individual reminder

### LINKED ISSUE #4: See Individual Reminder
When the user clicks on one of the reminders on the page, they are taken to an individual reminders route (ie: '/reminders/1', if the id of the reminder is `1`). The title of the reminder object should be displayed.

So, what needs to happen here?

* You'll need a `reminder` route in addition to the `reminders` route.
* It should be nested inside the `reminders` route.
* The user should still see all of the reminders from your previous feature. The specific reminder they are looking at now should be rendered in the outlet of the `reminders.hbs` template.
* There should be some CSS for the `.active` class.

There is a failing test set up in `tests/acceptance/reminder-list-test.js` to get you started - this test needs to be rewritten to reflect the route you are expecting to visit.

### PR DESCRIPTION
**Pull request by rcwestlake**
This code displays an individual reminder under the list of reminders. It also adds an active class to the selected reminder title. 

http://recordit.co/ovgAFs0Rw6

We passed the acceptance test for reminder-list-test.js

This closes #4 

@martensonbj @tman22 @stevekinney @nfosterky

### CODE REVIEW

File: app/router.js [hunk 1]
```diff
@@ -7,7 +7,9 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders');
+  this.route('reminders', { resetNamespace: true }, function() {
```
Comment (nfosterky) [inline app/router.js]: resetNamespace should not be necessary if you fix your `{{link-to...` reminder page link

File: app/templates/reminders.hbs [hunk 2]
```diff
@@ -1,7 +1,15 @@
 <h1>remEMBER</h1>
 <ul>
   {{#each model as |reminder|}}
-    <li class="spec-reminder-item">{{reminder.title}}</li>
+    <li class="spec-reminder-item">
+      <h3>
+        {{#link-to 'reminder' reminder class="spec-reminder-title" }}
```
Comment (nfosterky) [inline app/templates/reminders.hbs]: Check out the [docs](https://guides.emberjs.com/v2.10.0/templates/links/#toc_the-code-link-to-code-helper) on how to `{{link-to...` nested routes

**Comment by rcwestlake** [comment]
Thanks @nfosterky. I figured it out and will push up later tonight or in the morning. FYI @hilarylewis92

**Comment by hilarylewis92** [comment]
@nfosterky Made changes for routes. See above.

File: app/router.js [hunk 3]
```diff
@@ -7,7 +7,9 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-  this.route('reminders');
+  this.route('reminders', function() {
```
Comment (Tman22) [inline app/router.js]: Good fix

**Pull request merged**